mobile/native/src/config/api.rs

```diff
@@ -1,4 +1,5 @@
 use crate::config::ConfigInternal;
+pub use bdk::bitcoin::Network;
```

Comment:
`pub use` of internal library is a bit concerning, especially when we have a custom type for it.

it would be easier if the`Network` that we expose (our type) could parse a string itself.